GeoFire/API/GeoFire.h

```diff
@@ -70,7 +72,7 @@ typedef void (^GFCallbackBlock) (CLLocation *location, NSError *error);
  * @param location The location as a geographic coordinate
  * @param key The key for which this location is saved
  */
-- (void)setLocation:(CLLocation *)location
+- (void)setLocation:(nullable CLLocation *)location
```

Comment:
We don't want this nullable as `nil` is not a valid location.

Reply:
Note that the internal API `setLocationValue:forKey:withBlock:` (https://github.com/firebase/geofire-objc/blob/master/GeoFire/Implementation/GeoFire.m#L84) allows location to be nil (when we delete things), but the public API probably shouldn't allow this.

Reply:
Ah yep, I was basing it off the internal API.